homework5/tests/task1_test.py

```diff
@@ -0,0 +1,38 @@
+from homework5.oop_1 import Homework, Student, Teacher
+
+
+def test_teacher_creation():
+    teacher = Teacher("Daniil", "Shadrin")
+    assert teacher.__getattribute__("last_name") == "Daniil"
+    assert teacher.__getattribute__("first_name") == "Shadrin"
```

Comment:
You could use a simple `instance.attribute` instead of calling magic methods.

Reply:
I write `getattr(student, "last_name")`

Reply:
Done